config/default.js

```diff
@@ -104,7 +104,7 @@ module.exports = {
     baseRetryDelay: 1000,
     retryCount: 3, // Number of retries for pay invoice failure
   },
-
+  useAddressFallback: false,
```

Comment:
Should this be under invoices, at `invoices.useAddressFallback`?

Reply:
absolutely